README.md

```diff
@@ -30,7 +30,7 @@ addon | version | maintainers | summary
 [stock_move_line_auto_fill](stock_move_line_auto_fill/) | 15.0.1.0.0 |  | Stock Move Line auto fill
 [stock_move_quick_lot](stock_move_quick_lot/) | 15.0.1.0.0 |  | Set lot name and end date directly on picking operations
 [stock_no_negative](stock_no_negative/) | 15.0.1.0.0 |  | Disallow negative stock levels by default
-[stock_owner_restriction](stock_owner_restriction/) | 15.0.1.0.0 |  | Do not reserve quantity with assigned owner
+[stock_owner_restriction](stock_owner_restriction/) | 15.0.1.0.1 |  | Do not reserve quantity with assigned owner
```

Comment:
This shouldn't be modified by you

Reply:
This is because you have included the commit `[UPD] addons table in README.md`. Probably a rebase over an outdated branch. Remember always to do `git fetch --all` to get latest sources of everything.

Reply:
ok @pedrobaeza , thanks